src/encoded/static/components/summary.js

```diff
@@ -0,0 +1,464 @@
+            document.getElementById(`${this.chartId}-legend`).innerHTML = chart.generateLegend();
+        }
+    }
+
+    createChart() {
+        const { statusData } = this.props;
+
+        // Initialize data object to pass to createBarChart.
+        const data = {
+            anisogenicDataset: null,
+            isogenicDataset: null,
+            unreplicatedDataset: null,
+            labels: experimentStatuses,
+        };
+
+        // Convert statusData to a form createBarChart understands.
+        let facetData = statusData.find(facet => facet.key === 'anisogenic');
+        data.anisogenicDataset = facetData ? generateStatusData(facetData.status.buckets, data.labels) : [];
+        facetData = statusData.find(facet => facet.key === 'isogenic');
+        data.isogenicDataset = facetData ? generateStatusData(facetData.status.buckets, data.labels) : [];
+        facetData = statusData.find(facet => facet.key === 'unreplicated');
+        data.unreplicatedDataset = facetData ? generateStatusData(facetData.status.buckets, data.labels) : [];
+
+        // Generate colors to use for each replicate type.
+        const colors = globals.replicateTypeColors.colorList(globals.replicateTypeList);
+
+        createBarChart(this.chartId, data, colors, globals.replicateTypeList, 'Replication', this.props.linkUri, (uri) => { this.context.navigate(uri); })
+            .then((chartInstance) => {
+                // Save the created chart instance.
+                this.chart = chartInstance;
+            });
+    }
+
+    render() {
+        const { totalStatusData } = this.props;
+
+        // Calculate a (hopefully) unique ID to put on the DOM elements.
+        this.chartId = 'status-chart-experiments';
+
+        return (
+            <div className="award-charts__chart">
+                <div className="award-charts__title">
+                    Status
+                </div>
+                {totalStatusData ?
+                    <div className="award-charts__visual">
+                        <div id={this.chartId} className="award-charts__canvas">
+                            <canvas id={`${this.chartId}-chart`} />
+                        </div>
+                        <div id={`${this.chartId}-legend`} className="award-charts__legend" />
+                    </div>
+                :
+                    <div className="chart-no-data" style={{ height: this.wrapperHeight }}>No data to display</div>
+                }
+            </div>
+        );
+    }
+}
+
+SummaryStatusChart.propTypes = {
+    statusData: PropTypes.array.isRequired, // Experiment status data from /summary/ search results
+    totalStatusData: PropTypes.number.isRequired, // Number of items in statusData
+    linkUri: PropTypes.string.isRequired, // URI of base link for each bar to link to
+};
+
+SummaryStatusChart.contextTypes = {
+    navigate: PropTypes.func,
+};
+
+
+// Render the horizontal facets.
+class SummaryHorzFacets extends React.Component {
```

Comment:
Avoid abbreviations in class and variable names.

Reply:
Just my own personal rule of thumb. It probably wasn't very ambiguous in this case, but generally I don't think saving a few bytes is worth making someone reading the code later (including future me) have to guess about what the abbreviation stands for.